microsoft-edge/devtools-guide-chromium/css/index.md

```diff
@@ -36,7 +36,7 @@ Follow the interactive tutorial sections in this article to learn the basics of
 
    In the **DOM Tree** panel under the **Elements tool**, the `Inspect Me!` element is highlighted.
 
-   ![The inspected element is highlighted in the DOM Tree](./index-images/css-elements-inspect-me.png)
+   ![The inspected element is highlighted in the DOM Tree](./index-images/inspect-me.png)
 
 1. In the `Inspect Me!` element, find the value of the `data-message` attribute and copy it.
 
```

Comment:
change line 43 from:
1. On the page view, enter the `data-message` value that you copied in the **Value of data-message:** text box.
to:
1. On the page view, enter the `data-message` value that you copied in the **Value of data-message** text box.
ie remove colon, like omit "..." that's in UI control label.